sklearn/model_selection/_validation.py

```diff
@@ -654,7 +659,7 @@ def _fit_and_score(estimator, X, y, scorer, train, test, verbose,
     return result
 
 
-def _score(estimator, X_test, y_test, scorer):
+def _score(estimator, X_test, y_test, scorer, error_score):
```

Comment:
Even if this is private API, introducing the new arg as a kwarg would ensure backward compat at no cost (it actually improves readability a bit I think).

```suggestion
def _score(estimator, X_test, y_test, scorer, error_score="raise"):
```